qopt/cost_functions.py

```diff
@@ -1071,23 +1073,27 @@ def __init__(self,
             print('The systematic errors must be neglected if no target is '
                   'set!')
             self.neglect_systematic_errors = True
+    
+    def _to_comp_space(self, dynamic_target):
+        """Map an operator to the computational space"""
+        if self.computational_states is not None:
+            dynamic_target = dynamic_target.truncate_to_subspace(
+                subspace_indices=self.computational_states,
+                map_to_closest_unitary=self.map_to_closest_unitary,
+                )
```

Comment:
Here is somewthing missing...

Reply:
so will you make changes to this function?

Reply:
Done :)